Modules.md

```diff
@@ -1,7 +1,10 @@
 # Modules
 
 The distributable, loadable, and executable unit of code in WebAssembly
-is called a **module**. At runtime, a module can be loaded by a runtime
+is called a **module**. In a C/C++ world, a WebAssembly module is not a
+single ```.o``` file or a single class implementation but rather the
+equivalent of a C/C++ program with all its ```.o``` files linked together. At
```

Comment:
This is true in the MVP, but won't be true once we have dynamic linking (a module would be equivalent to either an executable or dynamic-link-library).  The explanation of what an instance contains below mentions this.  At the end of this section, we talk about what a wasm module is from a Web perspective; perhaps you could move this sentence to the end to explain what a wasm module looks like from a C/C++ perspective (and add a bit about dynamic linking).